### ISSUE TITLE
Feature Request: Regeneration if it rains / while diving

### ISSUE THREAD

**Issue by TomberWolf** [thetorine/thirstmod#30]
It would be nice if the thirstbar fills up if it's raining and / or if you're diving in water.

**Comment by PitchBright** [comment]
I disagree, for the obvious reason that just because you get wet, doesn't mean you've consumed water... letalone enough water to counter dehydration. 

To give you a like example; it would be the equivalent of feature requesting hunger to be filled just because you're standing in a field of wheat.

**Comment by TomberWolf** [comment]
Sure, but Minecraft don't let you fold your hands to collect rain and drink it. ;-)

What if you're in a pond and would be able to press the right mouse-button to drink?

**Comment by PitchBright** [comment]
Yes, that makes much more sense.
